modules/ROOT/pages/_partials/include-alerts-config.adoc

```diff
@@ -1,31 +1,34 @@
-= Create an Advanced Alert
-//INCLUDED IN GRAPH CONFIG FILE (dashboard-custom-config-graph.adoc#create-an-advanced-alert), advanced-alert.adoc
+= Create an Alert for a Custom Dashboard
+
 [[alerts]]
 
 . Navigate to the configuration screen for a graph in a custom dashboard. For guidance, see xref:dashboard-custom-config.adoc[Configuring Custom Dashboards].
-. Provide a general configuration for the graph that (at minimum) defines the metric, environment, and resource for which you want to set the advanced alert. +
+. Provide a general configuration for the graph that (at minimum) defines the metric, environment, and resource for which you want to set the alert. 
++
 For guidance, see xref:dashboard-custom-config-graph.adoc#general_settings[General Settings].
 . Navigate to the *Alert* configuration tab for the graph.
```

Comment:
Below is the main difference. Everything else is basically changing the language in the doc from "advanced alert" to clarify that we're talking about alerts for charts in custom dashboards.